src/Models/Classes/3DVConfig.ts

```diff
@@ -34,12 +37,15 @@ export enum WidgetType {
     Link = 'Link',
     Value = 'Value',
     Trend = 'Trend',
-    Panel = 'Panel'
+    Panel = 'Panel',
+    DataHistory = 'Data history'
```

Comment:
this enum is used in builder components which is mapping to the "type" string of widgets in schema. There has been no enum for type of widget generated from schema. We need this enum since the value of the keys are used as labels on UI (e.g. the title of the widgets in WidgetDialog are the values of this WidgetType enum)